packages/quantic/force-app/main/default/lwc/descriptionStrengthIndicator/descriptionStrengthIndicator.js

```diff
@@ -0,0 +1,24 @@
+import { LightningElement, api } from 'lwc';
+
+export default class DescriptionStrengthIndicator extends LightningElement {
+    @api helpLabel = "Don’t know what to write?";
+    progression = 0;
+
+    get variant() {
+        return this.progression < 100 ? "warning" : "base-autocomplete";
+    }
+
+    get message() {
+        return this.progression === 0 ? "Provide details" : this.progression < 100 ? "Provide more details " : "Thank you!";
+    }
```

Comment:
I would make labels for these perhaps? even if this goes in the cookbook it would be interesting to have them be translatable. Look for other quantic components with labels to create the same pattern.

Reply:
I will update this 👌